6_3_iterable.js

```diff
@@ -0,0 +1,48 @@
+class Group {
+    constructor(array) {
+        this.array = array;
+    }
+
+    static from(array) {
+        let group = new Group(array);
```

Comment:
U don't have to do this assignment. U can just write return new Group(array). Furthermore u should avoid let and replace it with const if u dont change the reference.

Reply:
And do you need this method ? Actually new Group(arr) is the same as Group.from(array)

Reply:
I see that static method is a part of your exercise, but do you need same assignment in constructor in this case ?